### PR TITLE
docs: adding a gif of doing a highlight on the website

### LINKED ISSUE #110: Feature: Usage gifs
### Type of feature

🍕 Feature

### Current behavior

There is no guide on how to use the product and we need the following features explained via gif.

I will work on this for the 100daysofOSS

### Suggested solution

- [ ] Highlights
- [ ] Collaborations
- [ ] Insight Pages

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [x] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by CBID2**
## Description
This PR adds a GIF demonstrating how to post a highlight on OpenSauced without the Chrome extension.

## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [X] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes #110 
## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [X] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: static/gif/highlight-explain.gif
Comment (bdougie) [inline static/gif/highlight-explain.gif:1]: Can you create a new one with the loom timer and avatar?

https://gif.ski/ is an option.

**Review by bdougie — CHANGES_REQUESTED** [review]

File: static/gif/highlight-explain.gif
Comment (CBID2) [reply static/gif/highlight-explain.gif:1]: Made the necessary changes @bdougie

**Review by bdougie — CHANGES_REQUESTED** [review]
We need a gif in a smaller size. Consider zooming in on the screen.

**Comment by CBID2** [comment]
> We need a gif in a smaller size. Consider zooming in on the screen.

How's this one @bdougie:
highlights-demo

**Comment by bdougie** [comment]
> > We need a gif in a smaller size. Consider zooming in on the screen.
> 
> How's this one @bdougie: highlights-demo

Can you turn off the grammarly

**Comment by CBID2** [comment]
> > > We need a gif in a smaller size. Consider zooming in on the screen.
> 
> > 
> 
> > How's this one @bdougie: highlights-demo
> 
> 
> 
> Can you turn off the grammarly 

Sure

**Comment by CBID2** [comment]
> > > We need a gif in a smaller size. Consider zooming in on the screen.
> > 
> > 
> > How's this one @bdougie: highlights-demo
> >       
> >         [
> >           
> >         
> >             highlights-demo
> >           ](https://user-images.githubusercontent.com/105683440/261824363-6e544006-54cd-4167-baf5-4ed8f5489c02.gif)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://user-images.githubusercontent.com/105683440/261824363-6e544006-54cd-4167-baf5-4ed8f5489c02.gif)
> 
> Can you turn off the grammarly

How about this one @bdougie?
 
posting-highlight-demo

**Comment by CBID2** [comment]
> The current gif shows a lot of typing. Can you make the recorded action you pasting the PR URL and clicking the auto-summary button.
> 
> Think of this like a commercial with only 5 seconds to convince the suer to try this feature. We don't need to show the user how to type their own summary, but rather we can sell them on generating a summary.
How's this one @bdougie:  
highlights-demo

**Comment by bdougie** [comment]
This gif is much better.

**Comment by CBID2** [comment]
> This gif is much better.

Done

**Comment by CBID2** [comment]
Is this the PR you're revising @bdougie?

**Comment by bdougie** [comment]
No, but this gif is still big. Currently 2.6mb

**Comment by CBID2** [comment]
> No, but this gif is still big. Currently 2.6mb

Ok I'll find a way to shrink it @bdougie

File: static/gif/highlights-demo.gif
Comment (bdougie) [inline static/gif/highlights-demo.gif:1]: Looking at this and it is cropped on the wrong place. 

Screen Shot 2023-09-06 at 10 35 52 AM

**Review by bdougie — DISMISSED** [review]
cropping of gif makes it not viewable.

**Comment by CBID2** [comment]
> cropping of gif makes it not viewable.

Noted

**Comment by CBID2** [comment]
> > cropping of gif makes it not viewable.
> 
> Noted
How about now @bdougie: 
highlight

**Comment by bdougie** [comment]
> How about now @bdougie:

So much better

**Comment by CBID2** [comment]
> > How about now @bdougie:
> 
> So much better

Hey @bdougie. I made the change

**Review by BekahHW — APPROVED** [review]

**Pull request merged**